get_version.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
```

Comment:
```suggestion
set -eu 
```
set -e causes the shell to exit when an unguarded statement evaluates to a false value.
^ might be very significant in case of, for example, unsuccessful cd.
set -u causes the shell to exit with an error when an undefined variable is referenced.